src/transformers/models/big_bird/convert_bigbird_original_tf_checkpoint_to_pytorch.py

```diff
@@ -0,0 +1,59 @@
+# coding=utf-8
+# Copyright 2018 The HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Convert BigBird checkpoint."""
+
+
+import argparse
+
+from transformers import BigBirdConfig, BigBirdForMaskedLM, load_tf_weights_in_big_bird
+from transformers.utils import logging
+
+
+logging.set_verbosity_info()
+
+
+def convert_tf_checkpoint_to_pytorch(tf_checkpoint_path, big_bird_config_file, pytorch_dump_path):
```

Comment:
This script works perfectly! Good job :-) 

Running the command: 

```python
python ../transformers/src/transformers/models/big_bird/convert_bigbird_original_tf_checkpoint_to_pytorch.py --tf_checkpoint_path ./tf_checkpoint/ckpt/model.ckpt-0 --big_bird_config_file ./tf_checkpoint/config.json --pytorch_dump_path ./hf_ckpt
```

whereas the big_bird_config_file was generated with:

```python
from transformers import BigBirdConfig

config = BigBirdConfig()
```

works as expected nice!

Reply:
Thanks for reviewing!